code/modules/vore/eating/living_vr.dm

```diff
@@ -88,6 +88,8 @@
 		if(!soulgem)
 			soulgem = new(src)
 		return TRUE
+
+	AddElement(/datum/element/slosh)
```

Comment:
// Make sure you don't call ..() on this one, otherwise you duplicate work.
/mob/living/simple_mob/init_vore()

simple mob has it's own init vore

Reply:
Also, you are adding the element AFTER the returns, it neds to go to the TOP.